pkgs/by-name/li/lightway-client/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  autoconf,
+  automake,
+  libtool,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "lightway-client";
+  version = "0-unstable-2025-09-04";
+
+  src = fetchFromGitHub {
```

Comment:
Both client and server use the same source, and most of the time is actually spent compiling the lightway-core crate. Maybe it's ok to have both as one package. I checked the binary sizes, it should even be fine to put them in the same output.